lib/app_profiler/backend/stackprof.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require "stackprof"
+
+module AppProfiler
+  class StackprofBackend < Backend
+    NAME = "stackprof"
+    DEFAULTS = {
+      mode: :cpu,
+      raw: true,
+    }.freeze
+
+    AVAILABLE_MODES = [
+      :wall,
+      :cpu,
+      :object,
+    ].freeze
+
+    def run(params = {})
+      started = start(params)
+
+      yield
+
+      return unless started
```

Comment:
can simplify this is a bit, I think.

`yield unless started `

Reply:
FYI this file was copied over verbatim, was formerly "profiler.rb".